docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -549,6 +556,38 @@ v: 1.0, stage: prod   v: 1.1, stage: canary            v: 1.1, stage: canary
 v: 1.0                (none)                           v: 1.0
 ====================  ===============================  ====================
 
+Scaled Locality Weights
+***********************
```

Comment:
Should this be `----` etc. to indicate a linkable section?

Reply:
I had no idea this was the distinction between stars and dashes lol

Making it linkable makes sense to me, will update